lib/utilities/monitoringHandler.js

```diff
@@ -64,6 +64,36 @@ const httpResponseSizeBytes = new client.Summary({
     help: 'Cloudserver HTTP response sizes in bytes',
 });
 
+const quotaEvaluationDuration = new client.Histogram({
+    name: 's3_cloudserver_quota_evaluation_duration_seconds',
+    help: 'Duration of the quota evaluation operation',
+    labelNames: ['action', 'code', 'type'],
+    buckets: [0.0001, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5],
+});
+
+const utilizationMetricsRetrievalDuration = new client.Histogram({
+    name: 's3_cloudserver_utilization_metrics_retrieval_duration_seconds',
```

Comment:
do we need to differentiate both durations ?
i.e. once the utilization metrics have been read, quota evaluation is just a few ops, with no more network ops or complex computation?

Reply:
Yes, because we might do more than one call to scuba in a single evaluation loop, for example to remove the inflights after detecting a quota exceeded, this help shaving visibility on the internal call latencies, versus the quota evaluation logic durations (with different cases)